CSharpException/CSharpException/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CSharpException;
+double sideOne = Triangle.EnterSide();
+double sideTwo = Triangle.EnterSide();
+double sideThree = Triangle.EnterSide();
+
+Triangle triangleToExeptions = new(sideOne, sideTwo, sideThree);
+
+try
+{
+    if (sideOne > sideTwo + sideThree) { throw new SideValueException(sideOne, sideTwo, sideThree); }
```

Comment:
Move this verifications to separate method in Tringle class. Then call this method in try catch construction here.